src/AuditPage.php

```
@@ -226,41 +207,64 @@ private static function query_ai_posts(int $paged = 1): array
+    /**
+     * Resolve a valid YYYY-MM selection, defaulting to the current WordPress
+     * month for missing or crafted query parameters.
+     */
+    private static function selected_month(): string
+    {
+        $requested_month = '';
+        // phpcs:ignore WordPress.Security.NonceVerification.Recommended -- read-only filter, no action taken
+        if (isset($_GET['month']) && is_string($_GET['month'])) {
+            $requested_month = sanitize_text_field(wp_unslash($_GET['month']));
+        }
+        $parsed = \DateTime::createFromFormat('!Y-m', $requested_month);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DateTime' in method 'selected_month'. (undefined)

(StaticAccess)